### ISSUE TITLE
Consume the alarms to automatically shut down the clusters

### ISSUE THREAD

**Issue by denis-crsj** [kids-first/kf-portal-ui#2562]
### Feature
The goal here is to configure the alarms that were created so that they automatically shut down the clusters that are inactive for an hour

### Acceptance Criteria

**Scenario 1** 
1. Launch a new cluster and wait for an hour
3. The Cluster should shutdown automatically

**Scenario 2** 
1. Launch a new cluster
2. Lauch a notebook and run a spark job (like an sql query)
3. Wait for an hour
4. The Cluster should shutdown automatically

### Sub-Tasks
- [ ] Add a subscription to this topic : kf-api-variant-cluster-us-east-1-prd-sns : the SQS queue should be the subscriber (DevOps)
- [ ] Consume events from the SQS queue in kf-api-variant-cluster using scheduled tasks, like [cron](https://www.npmjs.com/package/cron) 

### Testing Strategy
Anybody with access to launch a cluster and a notebook

### Resources

_Include any links to additional documentation, technical and/or design specs_

**Comment by denis-crsj** [comment]
@evans-g-crsj I know we are still waiting on some devops testing/task from CHOP but let's close this as it is already released anyway and we can always reopen or open another ticket if we have an issue later on.
